tests/android/browser.spec.ts

```diff
@@ -63,7 +63,7 @@ test('androidDevice.launchBrowser should pass proxy config', async ({ androidDev
   });
   const context = await androidDevice.launchBrowser({ proxy: { server: `${loopback}:${server.PORT}` } });
   const page = await context.newPage();
-  await page.goto('http://non-existent.com/target.html');
+  await page.goto('http://nonexistent.com/target.html');
```

Comment:
this appears to be a violation of DNS rules. If you don't own the domain you shouldn't use it. You could use `nonexistent.example.com` or a Microsoft owned domain (see `*.contoso.com`).